src/review/bot/open_ai_interface.py

```diff
@@ -195,44 +273,81 @@ def generate_suggestions_with_source(
 
-    response = openai.ChatCompletion.create(
-        engine=OPEN_AI_MODEL,
-        messages=[
+            """,
+            },
+            {"role": "user", "content": file_src},
+            {"role": "assistant", "content": "Ready for the doc review."},
+            {
+                "role": "user",
+                "content": f"""Improve this module, adding numpydoc docstrings where missing.
+End all descriptions with a period. Keep sentences short and simple. Omit unnecessary commas.
+For method and function descriptions, start the docstring with a simple verb (no "s" or "es" at the end of the verb).
+For class descriptions, start the docstring with a verb ending in an "s" or "es".
+Place names of classes, methods, parameters, data objects, and commands in double backticks (``).
+Follow code in backticks with the noun describing whether this code represents (such as a method, function, parameter, or command).
+When specifying a default parameter value, use the format "The default is ..., in which case the ...." Place the default value in double backticks.
+When the default value is "None", format it as ``None``.
+Instead of "used to" before a verb, replace with "for" and the gerund form of the verb.
+Instead of "to be" before a verb ending in "ed", replace with "to" and the simple verb form (without the "ed".
+For a boolean parameter, start the description with "Whether ..."
+For a boolean return value, use the description "``True`` when successful, ``False`` when failed."
+For property descriptions, use only a noun string followed by a period.
+Replace the Latin phrase "e.g." with "for example" and the Latin phrase "i.e." with "that is". Begin a new sentence with "For example,". Do not include as part of the previous sentence.
+Replace "Retrieve", "Obtain", "Find", and "Return" with "Get".
+Spell ANSYS as Ansys, hfss as HFSS, edb as EDB, boolean as Boolean.
+Provide a summary of the changes that are made.""",
+            },
+        ]
+    elif file_src and not patch:
+        messages = [
+            {
+                "role": "system",
+                "content": """
+You are a GitHub review bot.  You first expect full filename. You then expect a patch from a GitHub pull request and you provide 'review items' to improve just the patch code using the context from the full source file. Do not include the line numbers in any code suggestions. There are 3 TYPEs of review items [GLOBAL, SUGGESTION, COMMENT]. Each review item must be in the format [<FILENAME>], [<LINE-START>(-<LINE-END>)], [TYPE], always between brackets: <Review text>
+
+Type: GLOBAL
+This must always included. This is a general overview of the file patch. If the file looks good, simply respond with "No issues found, LGTM!". Otherwise, indicate the kind of comments and suggestions that follow. Make this section short and do not include any line numbers (i.e., leave [<LINE-START>(-<LINE-END>)] empty.
```

Comment:
```suggestion
This must always be included. This is a general overview of the file patch. If the file looks good, simply respond with "No issues found, LGTM!". Otherwise, indicate the kind of comments and suggestions that follow. Make this section short and do not include any line numbers (i.e., leave [<LINE-START>(-<LINE-END>)] empty.
```